File: string2aheui.py
#!/usr/bin/python
#-*- coding: utf-8 -*-

import num2aheui

def processString(string, bord):
	if (len(string) == 0):
		return None

	nord = ord(string[0])
	r = ""
	if (bord == None):
		r = num2aheui.getLog(nord).getAheuiLog()
		r += u"빠맣"
	else:
		r = num2aheui.makeDiffNum(bord, nord).getAheuiLog()
		r += u"빠맣"

	nr = processString(string[1:], nord)
	if (nr == None):
		return r
	else:
		return r + nr

def main():
	num2aheui.createTable()

	while True:
		string = raw_input("please enter what you want - ").decode('utf-8')
		print processString(string, None) + u"하"

if __name__ == '__main__':
	main()

File: string2aheui.js
var string2aheui = function () {
	this.processString = function (n2a, str, bord) {
		if (str.length == 0) {
			return undefined;
		}

		var nord = str.charCodeAt(0);
		console.log(nord);
		var r = "";
		if (bord == undefined) {
			r = n2a.getLog(nord).getAheuiLog();
			r += "빠맣";
		} else {
			r = n2a.makeDiffNum(bord, nord).getAheuiLog();
			r += "빠맣";
		}

		var nr = this.processString(n2a, str.substring(1), nord);
		if (nr == undefined) {
			return r;
		} else {
			return r.concat(nr);
		}
	};
};